python/scripts/check_documentation.py

```diff
@@ -108,6 +113,12 @@ def get_python_default_model_name() -> str:
     return default_model_name
 
 
+def get_rust_default_model_name() -> str:
+    model_symlink_path = REPO_ROOT_DIR / "rust" / "gen" / "model"
+    assert model_symlink_path.is_symlink()
+    return model_symlink_path.readlink().name
```

Comment:
You don't need to step the directory? I would expect this to output `../../assets/models/standard_xx`

Reply:
I see, SG